docs/Project-Configuration/Project-Commands.md

```diff
@@ -334,6 +334,26 @@ Parameters:
 -   `files`: a list .xml files to parse and upload. Filepath globs can
     also be supplied to collect results from multiple files.
 
+## downstream_expansions.set
+
+downstream_expansions.set is used by parent patches to pass key-value
+pairs to child patches. This command no-ops for any requesters that are
+not patch, GitHub merge queue/pull request, or Evergreen legacy commit
+queue. The command takes the key-value pairs written in
```

Comment:
nit: The phrasing seems pretty clear but it might be a bit more obvious to rephrase to first state the affirmative rather than the negative. For example, "This command only has an effect in manual patches, GitHub merge queue/legacy commit queue, and PRs. For all other versions, it is a no-op.".